src/CustomPasswordExpiredController.js

```diff
@@ -30,7 +30,7 @@ function (Okta, FormController, FormType, Footer) {
         var expiringSoon = this.options.appState.get('isPwdExpiringSoon'),
             numDays = this.options.appState.get('passwordExpireDays');
         if (expiringSoon && numDays > 0) {
-          return Okta.loc('password.expiring.title', 'login', [numDays]);
+          return Okta.loc('oie.password.expiry.warning.title', 'login', [numDays]);
```

Comment:
Should we revert this change ? because it looks like `oie.password.expiry.warning.title` is removed from login.properties file but I still see `password.expiring.title`.

Reply:
🤦  I meant to use `password.expiring.title`